### DISCUSSION TITLE
Problem adding Databases.

### DISCUSSION THREAD

**Discussion by bo1658a** [pawelsalawa/sqlitestudio#4742] [Q&A] [answered]
### Details
Here you describe your problem details, or proposal for enhancement.

### Steps to reproduce
This applies only to bug reports. If this is a feature request, delete this section.

### Operating system
This applies only to bug reports. If this is a feature request, delete this section.

### SQLiteStudio version
This applies only to bug reports. If this is a feature request, delete this section.

**Comment by pawelsalawa** [comment]
How are you trying to add it and what exactly causes the problem?

**Comment by bo1658a** [comment]
I opened SQLite studio, went to Add Database, opened the folder and
selected the Database, clicked Open. But it doesn't show the database and I
see the following error message- [15:29:10] Cannot execute query on
undefined or invalid database.
Thanks,
Bob

**Comment by pawelsalawa** [comment, marked as answer]
It seems you did not add or did not open the database.

Do you see it added in the database list panel on the left? If so, did you double-click it to open it?

**Comment by bo1658a** [comment]
I cannot see it added in the database list panel on the left. I did double click it though.

**Comment by bo1658a** [comment]
I finally managed to double click on the Database from the Database view and I indeed saw the Database on the left list panel.
